hermes-schema/src/test/java/pl/allegro/tech/hermes/schema/schemarepo/SchemaRepoSchemaSourceClientTest.java

```diff
@@ -0,0 +1,240 @@
+package pl.allegro.tech.hermes.schema.schemarepo;
```

Comment:
I've been reusing old tests, transforming them to groovy shouldn't take long. :)